public/controllers/register-agent/containers/register-agent/register-agent.tsx

```diff
@@ -52,6 +51,11 @@ export const RegisterAgent = withReduxProvider(
       false,
     );
 
+    const appVersionMajorDotMinor = wazuhVersion
+      .split('.')
+      .slice(0, 2)
+      .join('.');
+
```

Comment:
**suggestions**: you can use the `PLUGIN_VERSION_SHORT` variable that is defined in `common/constants.ts` instead.

Reply:
Implemented in this [commit]([bcca0cc](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/bcca0cc0ecdaf6a1d3edf9d25458ecfef808af23))